### ISSUE TITLE
Problem reading manifest when target-path has changed

### ISSUE THREAD

**Issue by ABCurado** [fivetran/dbt-language-server#419]
Hi, thank for the extension!

On my `dbt_project.yml`
I am redirecting the target folder to /tmp
`target-path: '/tmp'`

And I get the following error in the extension output
```
dbt 2022-09-20T10:06:39.296Z:  Failed to read manifest.json Error: ENOENT: no such file or directory, open 'tmp/manifest.json'
    at Object.openSync (node:fs:585:3)
    at Object.func [as openSync] (node:electron/js2c/asar_bundle:5:1812)
    at readFileSync (node:fs:453:35)
    at readFileSync (node:electron/js2c/asar_bundle:5:9160)
    at a.parse (/Users/user/.vscode/extensions/fivetran.dbt-language-server-0.11.0-darwin-x64/server/out/webpack:/dbt-language-server/src/manifest/ManifestParser.ts:40:21)
    at p.updateManifestNodes (/Users/user/.vscode/extensions/fivetran.dbt-language-server-0.11.0-darwin-x64/server/out/webpack:/dbt-language-server/src/FileChangeListener.ts:95:63)
    at p.onInit (/Users/user/.vscode/extensions/fivetran.dbt-language-server-0.11.0-darwin-x64/server/out/webpack:/dbt-language-server/src/FileChangeListener.ts:44:10)
    at t.LspServer.initializeHandler [as onInitialize] (/Users/user/.vscode/extensions/fivetran.dbt-language-server-0.11.0-darwin-x64/server/out/webpack:/dbt-language-server/src/LspServer.ts:119:29)
    at requestHandler (/Users/user/.vscode/extensions/fivetran.dbt-language-server-0.11.0-darwin-x64/server/out/webpack:/dbt-language-server/node_modules/vscode-languageserver/lib/common/server.js:686:26)
    at /Users/user/.vscode/extensions/fivetran.dbt-language-server-0.11.0-darwin-x64/server/out/webpack:/dbt-language-server/node_modules/vscode-jsonrpc/lib/common/connection.js:459:41 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'tmp/manifest.json'
}
```

Seems like it is removing the first `/` from the manifest path.

If I move the the target-path to the default one it work flawlessly

Thank you!

**Comment by pgrivachev** [comment]
Hi, thank you for reporting this issue. I reproduced it and will work on it soon.

**Comment by pgrivachev** [comment]
Hey @ABCurado,

I published new release v0.11.1 with fix. Could you please update extension and check if it works on your side?

**Comment by ABCurado** [comment]
Just tested and it working! That was quick, thank you for the attention!
